src/client/Microsoft.Identity.Client/Internal/JsonWebTokenConstants.cs

```diff
@@ -31,13 +24,28 @@ internal class ReservedClaims
             public const string JwtIdentifier = "jti";
         }
 
-        internal class ReservedHeaderParameters
+        internal static class ReservedHeaderParameters
```

Comment:
>static [](start = 17, length = 6)

does this have to static?

Reply:
It should be since it's just a class holder for consts.